src/client/telemetry/TelemetryConstants.ts

```diff
@@ -0,0 +1,14 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+// Event Names
+export const PORTAL_WEBPAGE_PREVIEW = 'portalWebPagePreview';
```

Comment:
prefer to put telemetry string literals and values inline to simplify code usage.
More often than not, eventNames are used in usually one place in a codebase. the extra leap of going to a constant makes code and reviewing a lot harder to read and reason about.